src/app/debug/table/table.component.html

```diff
@@ -84,30 +82,30 @@ <h4 class="progress-report-timout">[One or more reports are in progress for more
         <mat-spinner mode="indeterminate" [diameter]="30"></mat-spinner>
       </div>
     }
-    @if (doneRetrieving) {
+    @else {
       <div data-cy-debug="table" class="table-responsive" id="metadataTable">
         <table class="table mb-0" matSort (matSortChange)="helperService.sortData($event, tableSettings.reportMetadata)" [style.font-size]="getFontSize()">
           <thead id="table-header">
             <tr>
               <th class="table-row-checkbox header-padding">
                 <input data-cy-debug="selectAll" type="checkbox" class="vertical-middle" [checked]="allRowsSelected" (click)="selectAllRows()" title="Select all rows" [style.width]="getCheckBoxSize()" [style.height]="getCheckBoxSize()">
               </th>
-              @for (header of viewSettings.currentView.metadataLabels; let i = $index; track header) {
+              @for (header of viewSettings.currentView.metadataLabels; let index = $index; track header) {
                 <th class="table-id-cell header-cell header-padding"
-                    [attr.mat-sort-header]="i"
+                    [attr.mat-sort-header]="index"
                     [title]="header">
                   {{ getShortenedTableHeaderNames(header) }}
                 </th>
               }
             </tr>
           </thead>
           <tbody data-cy-debug="tableBody">
-            @for (row of tableSettings.reportMetadata.slice(0, tableSettings.displayAmount); let i = $index; track row) {
+            @for (row of tableSettings.reportMetadata.slice(0, tableSettings.displayAmount); let index = $index; track row) {
               <tr class="table-row"
-                  (click)="highLightRow(i);"
-                  [ngClass]="{'highlight': selectedRow === i}"
+                  (click)="openSelectedReport(row.storageId, index);"
+                  [ngClass]="{'highlight': selectedRow === index}"
                   [style.background-color]="getStatusColor(row)"
-                  [attr.data-cy-record-table-index]=i
+                  [attr.data-cy-record-table-index]=index
```

Comment:
```suggestion
                  [attr.data-cy-record-table-index]="index"
```

Reply:
Weird that there weren't any there quotes to begin with. I added the double quotes.